pkgs/development/python-modules/dpath/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "dpath";
+  version = "1.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0gr7816pnzbyh9h1ia0qz0q1f9zfzacwb8dc36js8hw8x14myqqg";
+  };
+
+  meta = {
```

Comment:
If you're going to add a new package, it's generally expected that you'll add yourself to the maintainers.

Also, we usually try to write the meta like this:
```
meta = with stdenv.lib; { 
    ... 
}
```
```